aws-resources/src/main/java/io/opentelemetry/contrib/aws/resource/EcsResource.java

```diff
@@ -192,9 +192,8 @@ static void parseResponse(
           DockerImage parsedImage = DockerImage.parse(value);
           if (parsedImage != null) {
             attrBuilders.put(CONTAINER_IMAGE_NAME, parsedImage.getRepository());
-            // TODO: CONTAINER_IMAGE_TAG has been replaced with CONTAINER_IMAGE_TAGS
             attrBuilders.put(
-                io.opentelemetry.semconv.ResourceAttributes.CONTAINER_IMAGE_TAG,
+                io.opentelemetry.contrib.aws.resource.IncubatingAttributes.CONTAINER_IMAGE_TAGS,
```

Comment:
@srprash @wangzlei the previously deprecated `io.opentelemetry.semconv.ResourceAttributes` was removed in the latest semconv (RC) artifact, and the deprecated attributes in `io.opentelemetry.semconv.incubating.ContainerIncubatingAttributes` don't go back far enough to still include the deprecated `container.image.tag`, so I went ahead and updated it to `container.image.tags`